simulation/sub8_gazebo/launch/duck.launch

```diff
@@ -40,6 +40,27 @@
     -x -22.3 -y 14.16 -z -4.47 -Z -1.57 -model lid"
     output="screen" />
 
+  <!-- Attempting to spawn the start gate -->
+  <node
+      name ="start_gate" pkg="gazebo_ros" type="spawn_model"
+      args="-file $(find sub8_gazebo)/models/start_gate/start_gate.sdf -sdf
```

Comment:
That's not a bad idea. I don't think there's anything in `sub8_simulation` that would be used in the future (but do look into it).

lol Then again, this doesn't actually help anybody.